components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIConsumerImpl.java

```diff
@@ -1409,7 +1409,7 @@ public int compare(Tag o1, Tag o2) {
             Map<String, Tag> tagsData = new HashMap<String, Tag>();
             try {
             	PrivilegedCarbonContext.getThreadLocalCarbonContext().setUsername(((UserRegistry)userRegistry).getUserName());
-                if (requestedTenant != null && !MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals(requestedTenant)) {
+                if (requestedTenant != null ) {//fix https://github.com/wso2/product-apim/issues/3449
```

Comment:
Don't need to have "//fix https://github.com/wso2/product-apim/issues/3449" in the code. having it in commit comment is enough.